lib/private/Files/Type/Detection.php

```diff
@@ -23,6 +23,7 @@ class Detection implements IMimeTypeDetector {
 	private const CUSTOM_MIMETYPEMAPPING = 'mimetypemapping.json';
 	private const CUSTOM_MIMETYPEALIASES = 'mimetypealiases.json';
 
+	/** @var array<string, array{string, string|null}> */
```

Comment:
Should only have consecutive numeric keys, no?

```suggestion
	/** @var array<string, list{string, string|null}> */
```